doc/building_blocks/plotting.rst

```diff
@@ -57,6 +61,10 @@ different heuristics to find cutting coordinates.
                      |hack|
                      Glass brain visualization
 
+|plot_connectome|    :func:`plot_connectome`
+                     |hack|
+                     Plotting a connectome
+
```

Comment:
I would put that below plot_stat_map: it's less likely to be what the user is looking for.